sdk/messaging/azwebpubsub/client_custom.go

```diff
@@ -139,11 +149,16 @@ func (c *Client) GenerateClientAccessUrl(ctx context.Context, options *GenerateC
 	}, nil
 }
 
+const DefaultExpirationTime = time.Hour
```

Comment:
If it's public we need a docstring. Or make it private.

And usually we put constants on top of file.